lumicks/pylake/kymotracker/detail/gaussian_mle.py

```diff
@@ -141,30 +146,38 @@ def gaussian_mle_1d(
     if fixed_background is not None and fixed_background <= 0:
         raise ValueError("Fixed background should be larger than zero.")
 
+    initial_position = x[np.argmax(photon_count)] if initial_position is None else initial_position
+    initial_position = np.atleast_1d(initial_position)
```

Comment:
I think it'd be cleaner if we just combine these two. Still would end up as multiple lines after `black` but it always feels icky to me to assign a variable and then immediately reassign

Reply:
Sure, no problem.